### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
Hello,

I have updated Readme.md.  Please review and suggest.

Live README.md link :  https://github.com/ApurvaSawant11/Dazzle-Ecommerce/blob/update-readme/README.md

### CODE REVIEW

**Review by purohitdheeraj — APPROVED** [review]
Awesome Work on UI
good to merge 🚀

File: README.md [hunk 1]
```diff
@@ -14,22 +14,43 @@ Dazzle Cerashop is an E-commerce website which sells ceramic products.
 
 ## Introduction
 
-- Dazzle Cerashop was built with an idea to sell products that has less harmful impact on environment.
+- Dazzle Cerashop is built with an idea to sell products that has less harmful impact on environment.
```
Comment (jalajcodes) [inline README.md]: Just a nitpick, but the grammatically correct sentence would be `...products that have less harmful....`

**Review by jalajcodes — APPROVED** [review]
Look good to merge.

File: README.md (hunk 1 above)
Comment (ApurvaSawant11) [reply README.md]: Ohh..yes Thanks! Fixed it

**Review by pradipgharge — APPROVED** [review]
Good job, UI looks very nice. Good to merge!

**Pull request merged**